src/utils/viewer/viewer.cpp

```diff
@@ -974,12 +974,15 @@ void Viewer::mouseReleaseEvent(MouseEvent& event) {
 }
 
 void Viewer::mouseScrollEvent(MouseScrollEvent& event) {
-  if (!event.offset().y()) {
+  // shift+scroll is forced into x direction on mac, seemingly at OS level, so
+  // use both x and y offsets.
```

Comment:
Too limit this weird workaround a bit, I'd suggest to wrap it in `#ifdef CORRADE_TARGET_APPLE`, at least.